packages/beacon-node/src/chain/chain.ts

```diff
@@ -1255,13 +1269,7 @@ export class BeaconChain implements IBeaconChain {
       if (!this.opts.supernode) {
         // Update custody requirement based on finalized state
         const validatorIndices = this.beaconProposerCache.getValidatorIndices();
-        const targetCustodyGroupCount = getValidatorsCustodyRequirement(headState, validatorIndices, this.config);
-        // only update if target is increased
-        if (targetCustodyGroupCount > this.custodyConfig.targetCustodyGroupCount) {
-          this.custodyConfig.updateTargetCustodyGroupCount(targetCustodyGroupCount);
-          this.logger.verbose(`Updated targetCustodyGroupCount=${this.custodyConfig.targetCustodyGroupCount}`);
-          this.emitter.emit(ChainEvent.updateTargetGroupCount, this.custodyConfig.targetCustodyGroupCount);
-        }
+        this.updateTargetCustodyGroupCount(getValidatorsCustodyRequirement(headState, validatorIndices, this.config));
```

Comment:
this conflicts a lot with https://github.com/ChainSafe/lodestar/pull/8105, would prefer getting that merged first as it's been reviewed already

Reply:
Ah, nice! will review and we can merge that first, will fix conflicts here after